Fix: API issue preventing multi-field updates (#875)
pluginsGLPI/fields: CHANGELOG.md

### Fixed
- Do not destroy the dropdown table/class if it is being used by another container.

> - Fixed a problem with fields being updated if there are several containers on the object.

Reviewer: ```suggestion
- Fix fields updates with multiple containers via the API
```
